recipes/_test.rb

```diff
@@ -18,15 +18,15 @@
 #
 # XXX: This recipe is only used for testing and should not be used otherwise.
```

Comment:
If this is a "test" recipe, shouldn't it live in a test cookbook under `tests/integration`?